src/storage/mvcc/reader/backward_scanner.rs

```diff
@@ -1203,6 +1203,8 @@ mod tests {
         assert_eq!(scanner.read_next().unwrap(), None);
     }
 
+    // This test runs slow, so we ignore it.
```

Comment:
How long will it takes?

Reply:
On my machine usually it takes xx seconds. We can re run test later with `cargo test -- --ignored`.

Reply:
But for most time, we don't run test that way. IMO, it is worthy of one minute, or it maybe neglects some bugs in the future.

Reply:
but bench only relates to performance not validity

Reply:
Hmm. You are right.